### ISSUE TITLE
[FE][SPIKE][DELIGHTER] How does the existing form system support image uploading?

### ISSUE THREAD

**Issue by mdewey** [department-of-veterans-affairs/va.gov-team#15045]
## User Story
As a veteran I should be able to upload a photo so that I can better inform my clinicians. 

## Related Documentation
 - [MVP: Visit Intro Initiative (Proof of Concept)](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/questionnaire/product/initiatives/visit-intro.md)

## Description
From user research, a trend came up that users want to upload photos. What exists today that we can use to make this possible. 

## Questions to Answer
> these are the targeted questions that need to be answered in this spike
- [ ] Are there any forms that upload pictures now? 
- [ ] Can we reuse their code?
- [ ] Are there any teams we can use as contacts?
- [ ] Where would this image end up? 

## Tasks
- [ ] Create a prototype
- [ ] Update relevant documentations

## Acceptance Criteria
- [ ] Questions are answered and documented in the team directory.

**Comment by mdewey** [comment]
Here are the first pass on the notes for how forms handle imagines. Its a little sparse and quick and dirty, but its the key points. 

tl;dr; it's been done in a few places already and code samples are plentiful.

@dillo, @kristenmcconnell , and @abuckley3385  Can you think of any other questions you have?

**Comment by kristenmcconnell** [comment]
It looks like there are clear UI examples on va.gov from the thread with Robin, which is good. I will check out those examples. My questions now mostly relate to the clinician's workflow and how adding images into what we are sending them will work.
